.github/workflows/dependabot-auto-merge.yml

```diff
@@ -0,0 +1,17 @@
+name: Dependabot auto-merge
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Enable auto-merge for Dependabot PRs
```

Comment:
Inspired by https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request

Do we want to auto-approve dependabot PRs too, to get a fully automated dependency management?

Reply:
I'm not sure about this 🤔 But I'm happy to try?

Reply:
I think we can do it for node libraries, but would be great to exclude things like Electron or in general libraries that can break the user experience without us noticing, because we don't have full tests for it

Reply:
> I think we can do it for node libraries, but would be great to exclude things like Electron or in general libraries that can break the user experience without us noticing, because we don't have full tests for it

That's a very valid concern 👍🏻 

Let's not auto-approve for now, see how many dependant PRs we are going to receive and whether reviews become an annoyance.

Reply:
sgtm!